spring-integration-test/src/test/java/org/springframework/integration/test/context/MockIntegrationContextTests.java

```diff
@@ -0,0 +1,44 @@
+package org.springframework.integration.test.context;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.config.EnableIntegration;
+import org.springframework.integration.dsl.IntegrationFlow;
+import org.springframework.integration.dsl.context.IntegrationFlowContext;
+import org.springframework.integration.endpoint.AbstractEndpoint;
+import org.springframework.integration.gateway.MessagingGatewaySupport;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
+
+@SpringJUnitConfig
+@ContextConfiguration(classes = MockIntegrationContextTests.Config.class)
+@SpringIntegrationTest
+public class MockIntegrationContextTests {
```

Comment:
Let's just remove this new void class altogether!
I'm really not sure that registering beans in  `start()/stop()` is OK idea, so I don't want to promote such a pattern with our tests.
Plus the fix is really simple and it is obvious that it is going to work in your project.